Core/src/IfcGeometryConverter/PlacementConverter.h

```diff
@@ -779,20 +781,71 @@ namespace OpenInfraPlatform {
 					//			WR2: (SIZEOF(PartOfU) = 1) XOR(SIZEOF(PartOfV) = 1) XOR(SIZEOF(PartOfW) = 1);
 					//	END_ENTITY;
 
-					std::vector<carve::geom::vector<2>> axis;
+					std::vector<carve::geom::vector<2>> axisVector;
+					carve::math::Matrix gridPositionMatrix;

```

Comment:
Set this to the default of `IDENT()` - identity matrix.

Reply:
Resolved: f1d242b